### PR TITLE
chore: auto update js code samples after package deploy

### PR DESCRIPTION
**Pull request by polomani**

### CODE REVIEW

File: scripts/afterRelease.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+brew install hub
+yarn typedoc --json docs.json
+node scripts/postProcessDocs.js
+
+packageVersion=$(node -e "const fs = require('fs');\
```
Comment (ghost) [inline scripts/afterRelease.sh]: Replace this with `packageVersion=${jq .version ./package.json}`?

File: scripts/afterRelease.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const packageFile = fs.readFileSync('./package.json');\
+const packageJson = JSON.parse(packageFile.toString());\
+console.log(packageJson.version)")
+branchName="feat/updateSDKJsExamples$packageVersion"
```
Comment (ghost) [inline scripts/afterRelease.sh:5]: We can't get conflict here? So we only update when we release a new version?

File: scripts/afterRelease.sh (hunk 1 above)
Comment (polomani) [reply scripts/afterRelease.sh]: nice

File: scripts/afterRelease.sh (hunk 2 above)
Comment (polomani) [reply scripts/afterRelease.sh:5]: correct, we only update after we release. means, we won't have the same branch name

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**